lib/iris/tests/unit/coord_categorisation/test_add_categorised_coord.py

```diff
@@ -99,7 +99,6 @@ def setUp(self):
             'proleptic_gregorian': np.array(list(range(360, 367)) +
                                             list(range(1, 4))),
             'noleap': np.array(list(range(359, 366)) + list(range(1, 4))),
-            'julian': np.array(list(range(360, 367)) + list(range(1, 4))),
```

Comment:
@marqh ( @djkirkham ) Any idea why this has been removed?

Also, the `from cf_units import CALENDARS as calendars` is now no longer used.

There is also a disconnect between the calendars in `calendars` and the calendars that this test is covering ... might want to join that up somehow ...